cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -502,6 +503,40 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def use_node_token_lookup_index() {
+    profileQuery(
+      title = "Node label LOOKUP index example",
+      text = "In the example below, a node TOKEN LOOKUP index is available.",
+      prepare = Some(_ => executePreparationQueries(List(
+        "CREATE LOOKUP INDEX node_label_lookup_index FOR (n) ON EACH labels(n)",
+      ))),
+      queryText = "MATCH (person:Person) RETURN person",
+      assertions = {
+        p =>
+          assertEquals(42, p.size)
+
+          checkPlanDescription(p)("NodeByLabelScan")
+      },
+    )
+  }
+
+  @Test def use_rel_token_lookup_index() {
+    profileQuery(
+      title = "Relationship TOKEN LOOKUP index example",
```

Comment:
In line with the change above, I would then change it to
```suggestion
      title = "Relationship type LOOKUP index example",
```